src/effects/presets/effectxmlelements.h

```diff
@@ -1,4 +1,5 @@
 namespace EffectXml {
+const QString FileHeader("<?xml version=\"1.0\" encoding=\"utf-8\"?>\n");
```

Comment:
shouldn't these be prefixed with `k`? And add QStringLiteral?

Reply:
I added QStringLiteral in 25794c95117fcce8fc8755796304cff75075d284. Prefixing all these with `k` seems superfluous to me. If you insist, I can do it, but it seems unnecessary to me.